lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
```

Comment:
Sorry, can you comment on this? I don't understand how you'd like to modify the source.